### DISCUSSION TITLE
nf-test with the `google-batch` executor

### DISCUSSION THREAD

**Discussion by chuck-fsdx** [askimed/nf-test#282] [Questions] [answered]
I'm using nf-test 0.9.2, and I'm trying to execute a pipeline test with Google Batch as the executor.

My nf-test.config contains the following directive:
```
config {
    workDir "gs://workdirbucket/{my-project-name}"
    configFile "configs/nf-test-gcloud.conf"
```
and the referenced config file specifies the 'google-batch' executor:
```
process {
    executor = 'google-batch'
    machineType = "n2-*"
    errorStrategy = 'retry'
}
```

When I attempt to run my test, nf-test doesn't use the google bucket as a work directory.  Instead, it creates a local directory called "gs:", and then throws the following error after trying to run:

```
Executor `google-batch` requires a Google Storage bucket to be specified as a working directory -- 
Add the option `-w gs://<your-bucket/path>` to your run command line or specify a workDir in your config file 
-- Check script '/{my_project_name}/./modules/main-module/main.nf' at line: 41 or see
 '/{my_project_name}/gs:/workdirbucket/{my_project_name}/tests/aab93aca2daa2f460a73162f2a47aefc/meta/nextflow.log' file for more details
```

Is this a use case that is known to work, and I'm just messing up the settings somehow?  Or has no one else attempted to run nf-test with a google-batch executor yet?

**Comment by lukfor** [comment, marked as answer]
Thank you for reporting this. Cloud executors are not yet supported at the moment, but we are working to implement them in the next release.

**Reply by chuck-fsdx** [reply]
Thanks for the reply!  Looking forward to that setup being supported.
